### PR TITLE
Merge from master to beta

### PR DESCRIPTION
**Pull request by ramya-stripe**
There have been more changes in the master branch since the last time we merged from it to beta. Doing on last merge from master before this week's beta releases

### CODE REVIEW

File: src/test/java/com/stripe/model/StandardizationTest.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
+package com.stripe.model;
+
```
Comment (jar-stripe) [inline src/test/java/com/stripe/model/StandardizationTest.java:2]: didn't this file get removed?  cc @prathmesh-stripe

File: src/main/java/com/stripe/net/RawRequestOptions.java [hunk 2]
```diff
@@ -48,11 +48,6 @@ public static RawRequestOptionsBuilder builder() {
   public static final class RawRequestOptionsBuilder extends RequestOptions.RequestOptionsBuilder {
     private Map<String, String> additionalHeaders;
 
-    /** Constructs a raw request options builder with default values. */
```
Comment (jar-stripe) [inline src/main/java/com/stripe/net/RawRequestOptions.java:51]: do we not need this constructor?

Comment (ramya-stripe) [reply src/main/java/com/stripe/net/RawRequestOptions.java:51]: It wasnt in the master branch, so I assumed we don't need it here either

**Review by jar-stripe — APPROVED** [review]
LGTM

File: src/main/java/com/stripe/net/RequestOptions.java [hunk 3]
```diff
@@ -394,7 +394,7 @@ protected static String normalizeClientId(String clientId) {
   }
 
   protected static String normalizeStripeVersion(String stripeVersion) {
-    // null stripeVersions are considered "valid" and use Stripe.stripeVersion
+    // null stripeVersions are considered "valid" and use Stripe.apiVersion
```
Comment (prathmesh-stripe) [inline src/main/java/com/stripe/net/RequestOptions.java:397]: Shouldn't it be `Stripe.stripeVersion`? https://github.com/stripe/stripe-java/blob/fb280de8ed50a12a16bf3bee2e358db9e0034b46/src/main/java/com/stripe/Stripe.java#L30

Comment (ramya-stripe) [reply src/main/java/com/stripe/net/RequestOptions.java:397]: You are right. I assumed what was in master to be the source of truth - didnt check the actual property name

**Pull request merged**